arci-urdf-viz/src/client.rs

```diff
@@ -133,6 +133,19 @@ impl UrdfVizWebClient {
                 let send_joint_positions_target =
                     { send_joint_positions_target_arc_mutex.lock().unwrap().take() };
                 if let SendJointPositionsTargetState::Some(target) = send_joint_positions_target {
+                    if target.duration.as_nanos() == 0 {
```

Comment:
workaround for "use of unstable library feature 'duration_zero'"

Reply:
Sorry!